test/cpp/lazy/test_lazy_ops.cpp

```diff
@@ -78,6 +80,18 @@ static inline at::DeviceType DefaultDevice() {
 
 }  // namespace
 
+TEST(LazyDynamicOpsTest, NarrowCopy) {
+  auto x = torch::rand({5, 10, 10}).to(kLazy);
+  const size_t Y_DIM = 3;
+  const size_t X_DIM_INDEX = 2;
+  auto y = torch::rand({Y_DIM}).to(kLazy);
+  auto ly = torch::lazy::TryGetLtcTensor(y);
+  auto dim_node = MakeNode<SizeNode>(ly->GetIrValue(), 0);
+  auto lmn = std::make_shared<torch::lazy::SymbolicIntNode>(dim_node);
+  auto z = x.narrow_copy(X_DIM_INDEX, 0, lmn->toSymInt());
+  AllClose(z.cpu(), x.cpu().narrow_copy(X_DIM_INDEX, 0, Y_DIM));
```

Comment:
So this tests that a symint can behave correctly in a static condition, and serves as a smoke test.  But i guess it doesn't really cover that dynamic aspects?

Reply:
yeah, though I could extend this and see how it behaves with different values, but this wasn't the goal of the milestone :)